pages/api/auth/[...nextauth].js

```diff
@@ -37,10 +39,27 @@ export default NextAuth({
         jwt: true,
         // maximum age of an idle session - 30 days
         maxAge: 30 * 24 * 60 * 60,
+        // update JWT on each login
+        updateAge: 0,
     },
     jwt: {
         // JWT secret used for generating a key
         secret: process.env.JWT_SECRET,
     },
-    // TODO: define custom callback functions
+    callbacks: {
+        session: async (session, user) => {
+            // Attach user id to session
+            session.id = user.id;
```

Comment:
do we want to add the role too?

Reply:
![image](https://user-images.githubusercontent.com/46411197/123666830-6b75c280-d7ee-11eb-90d6-761615f1fa97.png)
For adding the role, we have to check which relation field it has. For the relational field where there is a record, we map it back to an enum or some sort (TEACHER, PARENT, ADMIN, VOLUNTEER) and store it as `session.role`. What are your thoughts on this?

I think we can hold off on adding it until we have the create user endpoint so that this change is properly testable.

Reply:
Added task for this in backlog